pkgs/development/python-modules/pylibgen/default.nix

```diff
@@ -19,7 +21,7 @@ buildPythonPackage rec {
 
   checkInputs = [ pytest ];
 
-  # no tests in PyPI tarball
+  # too lazy to add pre-commit package
```

Comment:
what does this mean, we have `gitAndTools.pre-commit` make you can use `toPythonModule`?

Reply:
there's a seperate pypi package with the same name. I was trying to go home and just wanted to push the progress. I guess i didn't have to open a pr... xD